src/hooks/useFirestoreConnect.ts

```diff
@@ -15,23 +17,36 @@ export type Defined<T> = T & Exclude<T, undefined>;
  * This type allows us to automagically constrain the storeAs
  * parameter to keys that exist within ValidFirestoreKeys, ensuring
  * that we can't forget to define it in our types when using functions
- * that rely on this type (eg. useSparkleFirestoreConnect)
+ * that rely on this type (eg. useFirestoreConnect)
  *
  * @see ValidFirestoreKeys
- * @see useSparkleFirestoreConnect
+ * @see useFirestoreConnect
  * @see ReduxFirestoreQuerySetting
  */
-export interface SparkleRFQConfig extends ReduxFirestoreQuerySetting {
+
+export type SparkleUseFirestoreConfigObject = ReduxFirestoreQuerySetting & {
   storeAs?: ValidFirestoreKeys;
-}
+};
```

Comment:
Since you have done the same, I would think this should be a pattern now? We don't really need interfaces since it has basically become the same as the type

Reply:
I'm still not actually sure what the difference between the two styles is in this use case.. but happy to leave it with the `type` thing, as it seems to work.

Reply:
There was a difference in the previous TS version(you could only use interface to type objects AFAIK). Currently, the difference is strongly subjective

Reply:
Interesting.. maybe that's where I got the `interface` pattern from.

I think we seem to use this way more than the `interface` way, so let's say it's our chosen pattern until we run into a reason why it doesn't work.